src/components/Typography/HX/HX.stories.tsx

```diff
@@ -0,0 +1,128 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { customCopyable, customEllipsis } from '../Typography.mocks'
+import { HX } from './HX'
+import { Typography } from '..'
+import { loremIpsum } from '../../../utils/loremIpsum'
+
+const meta = {
+  title: 'Components/Typography/HX',
+  component: HX,
+  args: {
+    ...HX.defaultProps,
```

Comment:
it is a bit hard to work with copyable and ellipsis objects in the docs panel. Can we set them to boolean? It is true that we lose the customization, but we also have examples in later stories